main.go

```diff
@@ -13,10 +13,13 @@ import (
 func main() {
 
 	// Sorting the given slice
-	faktoren := []int{345, 78}
-
+	zahlen := []int{345, 78}
+	
+	
 	// Finding the time
```

Comment:
entferne den Kommentar `// Finding the time`

Reply:
hey duc ich habe es angepasst